pages/common/fossa.md

```diff
@@ -0,0 +1,24 @@
+# fossa
+
+> Fossa CLI - Fast, portable and reliable dependency analysis.
+> Homepage: <https://github.com/fossas/fossa-cli>.
+
+- Initialize a .fossa.yml configuration file:
+
+`fossa init`
+
+- Run a default project build:
+
+`fossa build`
+
+- Analyze built dependencies:
+
+`fossa analyze`
+
+- Generate reports:
+
+`fossa report`
+
+- Test current revision against FOSSA scan status and exit with errors if issues are found:
```

Comment:
What is meant by "current revision"? Is this the current revision of the project, or the version number of the `fossa` tool?

Also, small tweak :-)
```suggestion
- Test current revision against the FOSSA scan status and exit with errors if issues are found:
```